src/main/java/Tasks/Event.java

```diff
@@ -0,0 +1,21 @@
+package Tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    private final LocalDateTime at;
```

Comment:
I think 'at' is too short a name for a variable that has a rather large scope, should Event class become bigger in the future it might cause confusion in the future.
Since it might mean time or location.